tox.ini

```diff
@@ -40,3 +40,10 @@ basepython = python3.6
 deps =
     -r{toxinidir}/agent/requirements.txt
     -r{toxinidir}/agent/test-requirements.txt
+
+[testenv:alembic]
```

Comment:
I'm not sure I really understand this. We're not using `tox` in the deployed environment, and don't generally have a `git` tree; and we currently don't use `PostgreSQL` for unit testing. (Though I suppose we could, which might solve some of our `sqlite3` issues.) Not to mention that `alembic.run` doesn't currently actually *do* anything.

Is the idea to have the CI pipeline generate an Alembic version file during testing that somehow gets captured and used during container startup (when we actually have a PostgreSQL instance to initialize or update)? If so, how is this going to be fit in?

Reply:
To develop migrations you can't run `alembic revision` in a container environment as part of our process.